modules/Login/src/server/routes/loginRoutes.coffee

```diff
@@ -222,4 +222,14 @@ exports.getAuthors = (req, resp) ->
 		baseEntityServiceTestJSON = require '../src/javascripts/spec/testFixtures/BaseEntityServiceTestJSON.js'
 		resp.end JSON.stringify baseEntityServiceTestJSON.authorsList
 	else
-		csUtilities.getAuthors req, resp
+		if csUtilities.getAuthors?
```

Comment:
Is the point of this to preserve pre-existing CustomerSpecificServerFunctions at clients where we tie into an external authentication service? Just making sure I understand.